README.md

```diff
@@ -1,2 +1,147 @@
+
+- Create a local directory that you want to clone the repository.
+
+- On the terminal navigate to the folder you created and run this command: git clone https://github.com/Kashif-Saleem-Ghuman/LeaderBoard-by-Kashif.git
+
+- Now navigate to the repository folder using command prompt cd LeaderBoard-by-Kashif.
+
+- Make sure to intall linters using this link: https://github.com/microverseinc/linters-config.
+
+-Make sure to add project dependancies ( webpack)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Google chrome
+
+-Internet Explorer
+
+-Any broswer
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open the command prompt/terminal and navigate to the created directory.
+
+- On the terminal run this command git clone https://github.com/Kashif-Saleem-Ghuman/LeaderBoard-by-Kashif.git.
+
+- Go to the repository folder using command prompt cdTo-LeaderBoard-by-Kashif.
+
+
+- Make sure to intall linters using this link: https://github.com/microverseinc/linters-config.
+
+- Make sure to intall all dependencies.
+
+
+--->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👤**Authors** <a name="authors"></a>
+<h4>Kashif Saleem</h4>
+
+- GitHub: [@githubhandle](https://github.com/Kashif-Saleem-Ghuman)
+- Twitter: [@twitterhandle](https://twitter.com/Kashif14Saleem)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kashif-saleem-45ba95215/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Implement some UX improvements: To dynamically add and remove names and scores using the Leaderboard API.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Kashif-Saleem-Ghuman/To-do-list-by-Kashif/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+[Webpack](https://webpack.js.org/))
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/Kashif-Saleem-Ghuman/LeaderBoard-by-Kashif/blob/main/LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- **[OPTIONAL]** You did a great job documenting your project! 🚀 I suggest you remove this Note, as it is intended to be just a reminder to use a License in your project! 🤓 🛠️